lib/samson/hooks.rb

```diff
@@ -78,7 +78,9 @@ def callback(name, &block)
       end
 
       def view(name, partial)
-        hooks(name) << partial
+        hooks(name) do |view, *args|
```

Comment:
not sure I understand. Does `hooks(name)` return a method that will `yield`? So we are now storing the method itself rather than a string pointing to a view?